lib/narn-lib.js

```diff
@@ -17,13 +25,12 @@ exports.detectYarn = function detectYarn() {
 exports.getYarnArgs = narnArgs => narnArgs;
 
 exports.getNpmArgs = narnArgs => {
-  const yarnArgs = minimist(narnArgs);
+  const yarnArgs = minimist(reverseOption(narnArgs));
```

Comment:
Instead of reverseOptions - let's use [minimist's `opts.boolean`](https://www.npmjs.com/package/minimist#var-argv--parseargsargs-opts) option.

```suggestion
  const yarnArgs = minimist(narnArgs, {boolean: ['dev', 'D']});
```

This will make it so that minimist doesn't think the `--dev` is connected to the argument after it.